js/main.js

```diff
@@ -5,9 +5,51 @@ var MIN_MAP_Y = 130;
 var MAX_MAP_Y = 630;
 var adsArray = [];
 var mapBlock = document.querySelector('.map');
+var mapFilters = document.querySelector('.map__filters');
+var elementsMapFilters = mapFilters.querySelectorAll('input, select, fieldset');
+var adForm = document.querySelector('.ad-form');
+var elementsAdForm = adForm.querySelectorAll('fieldset');
 var mapPin = document.querySelector('.map__pin--main');
 var mapWidth = mapBlock.offsetWidth;
 var pinWidth = mapPin.offsetWidth;
+var pinHeight = mapPin.offsetHeight + 22;
+
+// Статус формы объявления
+function statusAdForm(element, isActive) {
+  for (var i = 0; i < element.length; i++) {
+    if (isActive) {
+      element[i].removeAttribute('disabled');
+    } else {
+      element[i].setAttribute('disabled', '');
+    }
+  }
+}
+// Статус Фильтров карты
+function statusMapFilters(element, isActive) {
```

Comment:
```suggestion
function statusMapFilters(elements, isActive) {
```

Массивы должны быть существительными во множественном числе. Так у человека который будет читать твой код будет больше контекста, чтобы понять что именно он (код) делает